### DISCUSSION TITLE
remote machine

### DISCUSSION THREAD

**Discussion by lguarda** [wojciech-kulik/xcodebuild.nvim#196] [General]
Hello first thanks for this wonderful work,
Than I Didn't see it written anywhere maybe because it's obvious,  since xcodebuild etc command are use, this plugin is for Mac only, but since only commands are used what would it take, or simply would it even be possible to make this plug-in partialy support other os like linux or windows, by making command ran through ssh, within a remote macos?

**Comment by wojciech-kulik** [comment]
Everything is possible :), it's just the question of how much time is required to do that :D.

Why don't you just connect to your macOS machine over SSH and run Neovim within your session?
